atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -91,9 +98,12 @@ def run(self):
                 new_image = image
             else:
                 new_image = self._pull_and_tag_image(image, build_json, str(nonce))
+            self.workflow.builder.recreate_parent_images()
             self.workflow.builder.parent_images[parent] = new_image
 
             if is_base_image:
+                if organization:
+                    self.workflow.builder.original_base_image.enclose(organization)
```

Comment:
Let's add a comment explaining why this is needed.

Reply:
ok